core/includes/image.inc

```diff
@@ -162,6 +164,18 @@ function image_get_info($filepath, $toolkit = FALSE) {
     if (isset($details) && is_array($details)) {
       $details['file_size'] = filesize($filepath);
     }
+    elseif (image_is_svg($filepath)) {
+      // Dimensions could be FALSE if the SVG is invalid, although we should
+      // have caught that on upload.
+      $dimensions = image_get_svg_dimensions($filepath);
+      $details = array(
+        'width'     => isset($dimensions['width']) ? $dimensions['width'] : 0,
```

Comment:
Sorry, but `0` seems like a weird fallback to me.

Reply:
Just a formal thing, but can we just not indent with multiple spaces here? I don't think, it's a forbidden thing, but appears to be uncommon in core code.

Reply:
We're setting this weird `0` in two places, now. But as it's about dimensions, why not setting some sane defaults inside that utility function and from then on rely on those?